Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -141,18 +141,28 @@ public override IEnumerable<Guid> ListChainIds()
         /// <inheritdoc/>
         public override void DeleteChainId(Guid chainId)
         {
+            _logger.Debug($"Deleting chainID: {chainId}.");
+            _lastStateRefCaches.Remove(chainId);
+
+            var cfName = chainId.ToString();
             try
             {
-                _lastStateRefCaches.Remove(chainId);
-
-                var cfName = chainId.ToString();
                 _chainDb.DropColumnFamily(cfName);
+            }
+            catch (KeyNotFoundException)
+            {
+                // Do nothing according to the specification: DeleteChainId() should be idempotent.
```

Comment:
It would be better if we can test if a `IStore` implementation actually satisfies this rule.

Reply:
The test seems to have already been added [here](https://github.com/planetarium/libplanet/pull/891/files#diff-5a20150ec899716c4ff9efea02fdf6a5R76).